packages/amplify-category-auth/commands/auth/update.js

```diff
@@ -70,6 +71,30 @@ module.exports = {
         }
         amplify.updateamplifyMetaAfterResourceUpdate(category, resourceName, 'customAuth', customAuthConfigured);
 
+        // Update Identity Pool dependency attributes on userpool groups
+        const allResources = context.amplify.getProjectMeta();
+        if(allResources.auth && allResources.auth.userPoolGroups) {
+          let attributes;
```

Comment:
Perhaps it would be more optimal to see what is different in behavior:

```
const attributes = ['UserPoolId', 'AppClientIDWeb', 'AppClientID'];
 
if(authParameters.identityPoolName) {
    attributes.push ('IdentityPoolId');
}
```

Reply:
Done. I chose the previous style for better readability, but I guess your suggestion is more maintainable.